start_kafka.sh

```diff
@@ -28,6 +28,9 @@ else
   sed -r -i "s@^#?inter.broker.listener.name=.*@inter.broker.listener.name=$KAFKA_INTER_BROKER_LISTENER_NAME@g" "/opt/kafka/config/kraft/server.properties"
 fi
 
-uuid=$(/opt/kafka/bin/kafka-storage.sh random-uuid)
-/opt/kafka/bin/kafka-storage.sh format -t $uuid -c /opt/kafka/config/kraft/server.properties
+if [[ -z "$KAFKA_SKIP_STORAGE_FORMAT" ]]; then
```

Comment:
Checking for existence of `meta.properties` could be a better approach to avoid having to start kafka twice with different environment variables. 

```
if [ ! -f /tmp/kraft-combined-logs/meta.properties ]; then
```

Reply:
Could the above suggestion be worked into this PR? Thanks

Reply:
Will update it tomorrow.

Reply:
Added and did basic tests with volume mounts locally. Seems to behave ok. Had to bump Kafka version to 3.1.1 as 3.0.0 was causing build issues - probably artifact moved to archive.